ansible/tasks/migrate-db-name.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Aeternity binary exists
+  stat:
+    path: "{{ aeternity_bin }}"
+  register: binary
+  changed_when: false
+
+- name: Ping aeternity node
+  command: "{{ aeternity_bin }} ping"
+  register: ping
+  failed_when: no
+  changed_when: false
+  when: binary.stat.exists
+
+- name: Aeternity db exists
+  stat:
+    path: "{{ db_path }}/mnesia"
+  register: db
+  changed_when: false
+
+- name: Check if already migrated
+  find:
+    paths: "{{ db_path }}/mnesia"
+    patterns:
+      - "schema.DAT.backup"
+      - "schema.DAT.*.backup"
+  register: find_results
+  when:
+    - db.stat.exists
+
+- name: Stop aeternity node
+  command: "{{ aeternity_bin }} stop"
+  when:
+    - binary.stat.exists
+    - ping.stdout == "pong"
+    - db.stat.exists
+    - find_results.files|length == 0
+
+- name: Migrate db schema names
+  command: "{{ aeternity_bin }} rename_db {{ db_path }}"
```

Comment:
I see that, if this task run, it assumes that this command is present.

Reply:
This means that, if this PR merged to stable branch, stable branch becomes unable to deploy current stable release (v2.1.0) though becomes able to deploy to-be v2.2.0.